### PR TITLE
DEV-540 - Show who uploaded the statement from the management interface

### PR DESCRIPTION
**Pull request by rqk2k**
### General description
Show who and when it was uploaded into the statement management interface.

### Designs
N/A

### Testing instructions
- Open Conferences (https://www.cbddev.xyz/conferences/) page.
- Select one of the meetings (https://www.cbddev.xyz/conferences/2021-2022).
- Open the sessions page (https://www.cbddev.xyz/conferences/2021-2022/sessions) and select one of the sessions.
- Click on the edit button.
- The information about who edit and when should be displayed next to the Save/Close buttons.

## Checklist before merging
* [X]  Branch name / PR includes the related Jira ticket Id.
* [X]  Tests to check core implementation / bug fix added.
* [X]  All checks in Continuous Integration workflow pass.
* [ ]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s).
* [ ]  Documentation updated (README, CHANGELOG...) (if required)

### CODE REVIEW

File: app/components/meetings/sessions/edit-intervention-modal.vue [hunk 1]
```diff
@@ -213,6 +234,16 @@ function onOrganizationChange(o) {
...
 }
 
+function isUserKronos(id) {
+    const pattern = /^[a-fA-F0-9]{24}$/;
+
+    if(pattern.test(id)) {
+        return true;
+    } else {
+        return false;
+    }
```
Comment (sbilodeau) [inline app/components/meetings/sessions/edit-intervention-modal.vue:244]: `pattern.test()` already return a boolean value. the `if` condition is useless
```suggestion
    return pattern.test(id);
```

File: app/components/meetings/sessions/edit-intervention-modal.vue [hunk 2]
```diff
@@ -113,6 +113,25 @@
                     </div>
                 
                     <div class="modal-footer">
+                        <div :v-if="meta" class="w-100">
+                            <small>
+                                Created by 
+                                <a 
+                                    v-if="isUserKronos(this.meta.createdBy.id)"
+                                    :href="`https://cbd.kronos-events.net/organizations/000000000000000000000000/contacts/${encodeURIComponent(this.meta.createdBy.id)}`"
```
Comment (blaisefonseca) [inline app/components/meetings/sessions/edit-intervention-modal.vue:121]: better to come from env variable `https://cbd.kronos-events.net`

File: app/components/meetings/sessions/edit-intervention-modal.vue [hunk 3]
```diff
@@ -113,6 +113,25 @@
...
+                                    target="_blank">{{ this.meta.createdBy.name }}</a>
+                                <span v-else>{{ this.meta.createdBy.name }}</span> 
+                                on {{ formatDate(this.meta.createdOn, 'yyyy-LL-dd HH:mm:ss') }}<br />
+
+                                Updated by
+                                <a 
+                                    v-if="isUserKronos(this.meta.updatedBy.id)"
+                                    :href="`https://cbd.kronos-events.net/organizations/000000000000000000000000/contacts/${encodeURIComponent(this.meta.updatedBy.id)}`"
+                                    target="_blank">{{ this.meta.updatedBy.name }}</a>
+                                <span v-else>{{ this.meta.updatedBy.name }}</span>
```
Comment (blaisefonseca) [inline app/components/meetings/sessions/edit-intervention-modal.vue:131]: maybe we can add link to accounts.cbd.int/profile/{user-id}

Comment (sbilodeau) [reply app/components/meetings/sessions/edit-intervention-modal.vue:131]: @rqk2k proposed the same but not every have accoutn access and it's mainly our ppl. 
Sandra Needs to know Who in Kronos as she may have to contact the persone

File: app/components/meetings/sessions/edit-intervention-modal.vue [hunk 4]
```diff
@@ -213,6 +234,12 @@ function onOrganizationChange(o) {
     this.title              = `${o.name} ${(o.acronym||'') && `(${o.acronym})`}`;
 }
 
+function isUserKronos(id) {
```
[the remark this answers is not in the document]
Comment (sbilodeau) [reply app/components/meetings/sessions/edit-intervention-modal.vue:237]: I agree

File: app/components/meetings/sessions/edit-intervention-modal.vue (hunk 2 above)
Comment (sbilodeau) [reply app/components/meetings/sessions/edit-intervention-modal.vue:121]: Maybe but should not change for years! so it's fine for now

File: app/directives/articles/articles-accordion.vue [hunk 5]
```diff
@@ -29,7 +29,7 @@
                         <cbd-add-new-article v-if="showEditButton" :id="article._id" target="_self" class="btn btn-default pull-right"></cbd-add-new-article>
                         <button class="btn btn-info pull-right btn-print" @click="print('cardItem_'+article._id, article)" 
                             style="cursor:pointer" v-if="showPrint"><i class="fa fa-print"></i> Print</button>
-                        <div v-html="sanitizeHtml($options.filters.lstring(article.content, $locale))" class="ck-content"></div>
+                        <cbd-article :article="article" :show-edit="true" :hide-cover-image="true"></cbd-article>
```
Comment (blaisefonseca) [inline app/directives/articles/articles-accordion.vue:32]: why hide-cover-image in accordion?

Comment (rqk2k) [reply app/directives/articles/articles-accordion.vue:32]: I don't know from where this come from, I didn't touch anything on this page.

Comment (blaisefonseca) [reply app/directives/articles/articles-accordion.vue:32]: yes its bit confusing, I made that change i think

**Review by blaisefonseca — APPROVED** [review]

File: app/components/meetings/sessions/edit-intervention-modal.vue (hunk 2 above)
Comment (rqk2k) [reply app/components/meetings/sessions/edit-intervention-modal.vue:121]: is there any ENV variable already with this information?

**Pull request merged**